app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,15 +1,9 @@
-class Admin::DashboardController < ApplicationController
-  before_action :require_admin
```

Comment:
Saw that you moved this to the newly created Admin::DashboardController - nice 👍